CONTRIBUTING.md

```diff
@@ -43,7 +43,8 @@ Conduct].
 
 Join following groups/channels to discuss ideas with other kpt contributors.
 
-1. Join our [email list](https://groups.google.com/forum/?oldui=1#!forum/kpt-dev)
+1. For developers join our [email list](https://groups.google.com/forum/?oldui=1#!forum/kpt-dev)
+1. If you'd like to be invited to all public meetings join [email list](https://groups.google.com/forum/?oldui=1#!forum/kpt-contribx)
```

Comment:
It can be super helpful to let users know what to expect from the public meetings. One doubt I got when talking with slack users is that they thought joining the meeting means they may need to speak, and have something prepared.

Reply:
Sounds great I added a couple more lines.